gobblin-api/src/main/java/gobblin/writer/DataWriterBuilder.java

```diff
@@ -25,11 +27,17 @@
  */
 public abstract class DataWriterBuilder<S, D> {
 
+  @Getter
```

Comment:
You can add the Getter annotation to the class declaration and it will be applied to all instance variables in the class. Saves the time of adding the annotation for each individual instance variable.